students/kpersic/module_3/blog/scss/_typography.scss

```diff
@@ -0,0 +1,40 @@
+h1,h2 {
+    color: $primary;
+}
+
+h1 {
+    font-size: 54px;
```

Comment:
Slično kao s marginama i paddingom, razmisli o primjeni tipografske skale (ako možeš). Više informacija ovdje:
- https://www.modularscale.com/
- https://type-scale.com/
- https://moderncss.dev/generating-font-size-css-rules-and-creating-a-fluid-type-scale/
- https://www.codementor.io/@ricardozea/100-responsive-typography-system-using-a-modular-scale-s5rhft58g